### PR TITLE
Added links to 'enlace_baja' and 'enlace_pertil_url' on news tematicas.

### PR DESCRIPTION
**Pull request by gponsu**

### CODE REVIEW

**Comment by Fernan2** [comment]
+1

File: app/views/news_tematica/news_tematicas/_unsubscribe.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %table{cellpadding: "0", cellspacing: "0", style: "width:100%;background-color:#f1f1f1; margin:0;color:#a9a9a9; font-size:11px;padding-bottom:5px;"}
   %tr
     %td{style: "vertical-align: middle;font-size:11px;padding:5px;"}
-      Para darte de baja, [enlace_desuscripcion_tematica]
+      |enlace_baja| - |enlace_perfil_url|
```
Comment (scruti) [inline app/views/news_tematica/news_tematicas/_unsubscribe.html.haml]: `[` or `|` ? That's the question. Could someone explain this to me :angel:

Comment (Fernan2) [inline app/views/news_tematica/news_tematicas/_unsubscribe.html.haml]: Look at configura_sustituciones_sendgrid:
https://github.com/Soluciones/Rankia/blob/d2e94ba2237741b1ab1528bda26f264dcdb537aa/lib/mailer_module.rb

We're replacing `"|#{ key }|"`

Comment (scruti) [inline app/views/news_tematica/news_tematicas/_unsubscribe.html.haml]: Thanks!

**Comment by scruti** [comment]
:+1:

**Comment by d2bit** [comment]
:+1:

**Pull request closed without merging**